src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -0,0 +1,111 @@
+﻿// <copyright file="AWSLambdaUtils.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Amazon.Lambda.Core;
+using OpenTelemetry.Contrib.Extensions.AWSXRay.Trace;
+
+namespace OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation
+{
+    /// <summary>
+    /// Class for getting AWS Lambda related attributes.
+    /// </summary>
+    internal static class AWSLambdaUtils
+    {
+        internal const string ActivitySourceName = "Amazon.AWS.AWSLambdaInstrumentation";
+        private const string CloudProvider = "aws";
+        private const string AWSRegion = "AWS_REGION";
+        private const string AWSXRayLambdaTraceHeaderKey = "_X_AMZN_TRACE_ID";
```

Comment:
I think it will not cause confusion as the only xray trace key we can get in Lambda is from environment variable, so probably not need to specify it's from ev, unless we have other way to fetch the key, say from Lambda context, instead of ev. Could you provide more context on this?

Reply:
That's fine.